DEVELOPING.md

```diff
@@ -0,0 +1,113 @@
+# Documentation for Developers
+
+This document describes how to set up and develop Native Build Tools on your local machine.
+
+## Environment
+
+The project uses Gradle as its build system. At the very minimum, you should set `JAVA_HOME` to a [Gradle-compatible JDK](https://docs.gradle.org/current/userguide/compatibility.html).
+
+Some build tasks require a GraalVM JDK (e.g., tests). You should set `GRAALVM_HOME` to an appropriate GraalVM JDK.
+
+## IDE Setup
+
+The Native Build Tools repo is set up as a multi-project Gradle project, with the Maven and Gradle plugins declared as subprojects of the root project.
+To set the project up in your IDE (e.g., IntelliJ IDEA), import the root project and the IDE should automatically import the subprojects.
+
+## Building and Testing
+
+You can use the various commands in the [Gradle build lifecycle](https://docs.gradle.org/current/userguide/build_lifecycle.html) to build and test the project.
+Some examples (all executed from the root of the repository):
+
+```bash
+# Build all projects
+./gradlew assemble
+
+# Build only the native-gradle-plugin (for example)
+./gradlew :native-gradle-plugin:assemble
+
+# Build and run all tests
+./gradlew build
```

Comment:
See you in 10 days :)

Reply:
what's in 10 days? 😱

Reply:
😆 good call, we should be conscientious of the planet (and our users' time)